components/scream/src/physics/shoc/tests/shoc_diag_second_shoc_moments_tests.cpp

```diff
@@ -292,7 +298,9 @@ struct UnitWrap::UnitTest<D>::TestDiagSecondShocMoments {
     // Get data from cxx
     for (auto& d : cxx_data) {
       d.transpose<ekat::TransposeDirection::c2f>(); // _f expects data in fortran layout
-      diag_second_shoc_moments_f(d.shcol(), d.nlev(), d.nlevi(), d.thetal, d.qw, d.u_wind, d.v_wind, d.tke, d.isotropy, d.tkh, d.tk, d.dz_zi, d.zt_grid, d.zi_grid, d.shoc_mix, d.wthl_sfc, d.wqw_sfc, d.uw_sfc, d.vw_sfc, d.thl_sec, d.qw_sec, d.wthl_sec, d.wqw_sec, d.qwthl_sec, d.uw_sec, d.vw_sec, d.wtke_sec, d.w_sec);
+      diag_second_shoc_moments_f(d.shcol(), d.nlev(), d.nlevi(), d.thetal, d.qw, d.u_wind, d.v_wind, d.tke, d.isotropy, 
```

Comment:
Please upstream merge this branch. After that, shcol, nlev, and nlevi are no longer methods, just access them directly.